xbmc/platform/win10/storage/Win10StorageProvider.cpp

```diff
@@ -31,9 +31,9 @@ using namespace winrt::Windows::Devices::Enumeration;
 using namespace winrt::Windows::Foundation::Collections;
 using namespace winrt::Windows::Storage;
 
-::IStorageProvider* ::IStorageProvider::CreateInstance()
+std::unique_ptr<::IStorageProvider>::IStorageProvider::CreateInstance()
```

Comment:
So what is wrong here? UWP builds are failing. Is it the lack of spaces? The space was there and was removed by clang-format...

Reply:
ok so this clashes with `winrt::Windows::Storage::IStorageDevice`... had to waste time compiling for uwp to understand the reason. Pretty much why I didn't want to do this in the first place.